### DISCUSSION TITLE
Crashing on zoomed-out timeline

### DISCUSSION THREAD

**Discussion by peteyus** [keiwando/toonsquid#549] [Q&A] [answered]
My kiddo has been working on an animation and, playing around, zoomed the timeline out very far which caused the app to crash. Unfortunately, it now crashes any time they open up the project and open the timeline view. I've attached a crash report from the iPad, is there any way to reset the timeline scale? All other projects are working fine, and that one project works no problem with frame by frame advancement, but consistently crashes every time the timeline is opened.

I did note that this exception report mentions that the 19.95 wall time limit was exceeded, but that seems unrelated to the amount of time in the app (it will stay open much longer than that, and can crash much more quickly than that, the crash is instantaneous when the timeline pane is expanded)

Troubleshooting attempted: Cleared project cache, cleared all project crashes, force closed and relaunch application, reboot iPad. 

[Crash report](https://github.com/keiwando/toonsquid/files/14566301/ToonSquid-2024-03-11-194830.zip)

**Comment by keiwando** [comment, marked as answer]
The bug causing the crashes should now be fixed with the 1.10.9 update. Thanks for the patience!
(I'm assuming you had already received my e-mail last week containing the project file with the timeline zoom reset)
